mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/OnlineKitPvp.java

```diff
@@ -30,19 +30,22 @@
     private volatile int experience;
     private volatile int currentKillstreaks;
     private volatile int highestKillstreaks;
-    private volatile BigDecimal bounty;
+    private volatile BigDecimal creditsBounty;
```

Comment:
You could make a map in case ariel wants 3 currencies. 4?

Reply:
I originally used a Map, but it was more messy and harder to see what the code was doing just by looking at it.

If we ever need more currencies, the implementation can change. In fact, the switch statement on the currency enum will fail to compile.